### DISCUSSION TITLE
Quartus 20.1 support

### DISCUSSION THREAD

**Discussion by epsilon-0311** [t-crest/patmos#94] [Q&A]
Hi all,

I started to develop a container image which provides all dependencies and has the compiler etc set up. With the setup script more or less everything worked fine, but when I try synthesizing patmos the Quartus fitter takes around 17 minutes on my AMD Ryzen 7 5800X, in comparison to Quartus 19.1 it only takes 7-8 seconds. 

Additionally when flashing the hello puts example onto my DE2-115 Board the process exits with an error, using 19.1 the program exits normal.

That behaviour is quite strange for me, and I'm a little lost looking for the error although Quartus 20.1 should still support the Cyclone IV FPGA line.

**Comment by Emoun** [comment]
Sounds very much like regression in Quartus 20.1. Have you looked at their open issues to see if they know, and if not, perhaps you should file a bug report.

Perhaps you can try experimenting with different boards (if you have access to others) and different designs to see if the issue is with the board or the Patmos core design.

I'm going to open an issue so we can keep track of this.

**Reply by epsilon-0311** [reply]
At least in intels knowledge base I didn't find anything related to this. I will create an issue there as soon as I have rebuilt my container and have created log files to compare. I will upload them to github to.

Nothing that is supported out of the box, but if I have time I will try it on my rz-easyfpga.

**Reply by Emoun** [reply]
I created  #95. Update it as you go with what you find out.
